pkgs/servers/passopolis/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, antBuild, fetchgit, git, python }:
+
+let
+  version = "b827b3a6176e050deb729009676fad7e86e5393a";
+in antBuild {
+  name = "passopolis-${version}";
+
+  src = fetchgit {
+    url = "https://github.com/WeAreWizards/passopolis-server";
+    sha256 = "0ywmymbjcfsxv1p1j0l0lw9cb7f79h23ic1c4b5w5nb0k9f4zvfq";
+    rev = "${version}";
+    leaveDotGit = true;
```

Comment:
It won't fail, but it will put garbage in the build.properties file. When running the ant jar command, it generates a build.properties file with the build time, git commit and description.